idaes/generic_models/properties/core/state_definitions/FPhx.py

```diff
@@ -132,7 +132,7 @@ def flow_mol_phase_comp_rule(b, p, j):
     b.phase_frac = Var(
         b.phase_list,
         initialize=1/len(b.phase_list),
-        bounds=(0, None),
+        bounds=(0, 1.001),
```

Comment:
Just wondering about the rationale to leave this at 0, where the other fracs were 1e-20.